velox/functions/sparksql/MightContain.cpp

```diff
@@ -33,26 +49,24 @@ class BloomFilterMightContainFunction final : public exec::VectorFunction {
     context.ensureWritable(rows, BOOLEAN(), resultRef);
     auto& result = *resultRef->as<FlatVector<bool>>();
     exec::DecodedArgs decodedArgs(rows, args, context);
-    auto serialized = decodedArgs.at(0);
     auto value = decodedArgs.at(1);
 
-    HashStringAllocator allocator{context.pool()};
-    VELOX_USER_CHECK(serialized->isConstantMapping())
-    BloomFilter output{StlAllocator<uint64_t>(&allocator)};
-    try {
-      auto sv = serialized->valueAt<StringView>(0);
-      output.merge(sv.data());
-    } catch (const std::exception& e) {
-      context.setErrors(rows, std::current_exception());
-      return;
+    if (!isSet_) {
```

Comment:
Please use bloomFilter_.isSet()

Reply:
what's the situation that it is not set?

Reply:
I think not set is for empty bloom filter. UT `nullBloomFilter` will run into this code path.

Reply:
You can set bloomFilter_.isSet() to const, I have another PR to set it to const